### PR TITLE
API docs

### PR DESCRIPTION
**Pull request by soderluk**
Add RAML specification for API docs
Automatically generate documentation and deploy to Github Pages

### CODE REVIEW

File: docs/api.raml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  description: API Health endpoints
+  get:
+    description: Health check endpoint
+    responses:
+      200:
+        body:
+          example: {}
+/fetch:
+  description: Fetch data from data source
+  post:
```
Comment (lietu) [inline docs/api.raml:36]: Do you think it's possible to document incoming headers here?

File: docs/api.raml [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                    "unit_measurement": "Celsius",
+                    "value": "22.5"
+                  }
+                }
+            signature:
+              description: Signature information
+              type: object
+              example: |
+                {
+                  "type": "<should match organizationPublicKeyURL in data product>",
```
Comment (lietu) [inline docs/api.raml]: The documentation is switched for `type` and `created`

Comment (soderluk) [reply docs/api.raml]: Good catch!

File: docs/api.raml (hunk 1 above)
Comment (soderluk) [reply docs/api.raml:36]: Yes, good that you noticed.

**Comment by lietu** [comment]
Looks great

**Review by lietu — APPROVED** [review]

**Pull request merged**